### ISSUE TITLE
How to resize the view of the presented controller ?

### ISSUE THREAD

**Issue by kevin-pics** [SebastianBoldt/Jelly#47]
```swift
let controllerA = ControllerA()
let navigationController = UINavigationController(rootViewController: controllerA)

let presentation = JellyFadeInPresentation()
let animator = JellyAnimator(presentation: presentation)
animator.prepare(viewController: navigationController)

self.present(navigationController, animated: true, completion: nil)
```

In controllerA, controllerB will be created and pushed to navigationController. My question is `How to resize controllerB's view ?` Thanks!

**Comment by nghuiqin** [comment]
I need same feature too. Figuring how to change the frame size of the presented viewcontroller but it won't work if I change the view frame size. It seems the layoutsubview will just read the size where we set at the beginning.

**Comment by ryandailey100** [comment]
Can this be implemented? Really need this working for my project. Thanks.

**Comment by SebastianBoldt** [comment]
Do I understand this correctly? 
You want to resize the presented viewController if it is already presented?

**Comment by SebastianBoldt** [comment]
@baby03201 & @ryandailey100 & @kevin-lxh

This feature is currently under development (already working like a charm) 
and will be available in version 2.0.0 next month 

🎉
